server/db.js

```diff
@@ -14,85 +14,94 @@ const config = {
   dropDatabase: true,
 };
 
-// schema of each individual review
-const reviewSchema = new mongoose.Schema({
-  id: mongoose.ObjectId,
-  user_icon: String,
-  review_text: String,
-  date: Date,
-  name: String,
-  rating: {
-    cleanliness: Number,
-    communication: Number,
-    check_in: Number,
-    accuracy: Number,
-    location: Number,
-    value: Number
-  }
-});
-
-db.on('error', console.error.bind(console, 'connection error:'));
-db.once('open', function() {
-  // populate the database once we are connected
-  initDB();
-});
-
 // get random number between min and max, inclusive
 const getRandomInt = (min, max) => {
   let range = max - min;
   return Math.floor(Math.random() * range) + min;
 }
 
-const random_names = ['Chris', 'Katie', 'Emmanuel', 'Josef', 'Kiara', 'Karen', 'Clarence', 'Jorge', 'Antonio',
+const randomNames = ['Chris', 'Katie', 'Emmanuel', 'Josef', 'Kiara', 'Karen', 'Clarence', 'Jorge', 'Antonio',
                       'Elana', 'Lim', 'Jake', 'James', 'Johnny', 'Jorgen', 'Haneen', 'Mataeux', 'Theo', 'Ryan',
                       'Jacob', 'Jenny', 'Alex', 'Alissa', 'Andrew', 'Anna', 'Arun', 'Adjoa', 'Billy', 'Brian',
                       'Carina', 'Catherine', 'Daniel', 'Erfan', 'Eric', 'Harris', 'Harrison', 'Jen', 'Jessie',
                       'Joel', 'Johnny', 'Joesph', 'Karl', 'Katharine', 'Liz', 'Mike', 'Minji', 'Mylani', 'Rebecca',
                       'Rob', 'Shaquon', 'Sophie', 'Sokhary', 'Susan', 'Victoria', 'Watson', 'Yas'];
 
-const random_text = ['apple', 'orange', 'pear', 'little', 'big', 'hack', 'reactor', 'review', 'great', 'the', 'a',
-                     'is', 'of', 'will', 'tree', 'normal', 'abstract', 'your', 'job', 'free', 'work', 'stay','home'.
+const randomText = ['apple', 'orange', 'pear', 'little', 'big', 'hack', 'reactor', 'review', 'great', 'the', 'a',
+                     'is', 'of', 'will', 'tree', 'normal', 'abstract', 'your', 'job', 'free', 'work', 'stay','home',
                      'apartment', 'kite', 'rent', 'stay', 'like', 'roof', 'room', 'bathroom', 'lorem ipsum'];
 
 const generateRandomReview = (length) => {
-  let review = [];
+  let review = '';
   for (let i = 0; i < length; i++) {
-    let index = Math.floor(Math.random(0, random_text.length));
-    review.push(random_text[index]);
+    let index = Math.floor(Math.random() * randomText.length);
+    // console.log(index);
+    review += randomText[index] + ' ';
   }
-}
+  review = review.slice(0, review.length - 1);
+  return review;
+};
 
 // generate a random document
 const generateRandomDocument = (id) => {
   const doc = {
-    id: id,
-    user_icon: 'google.com',
-    review_text: generateRandomReview(100),
+    userIcon: 'google.com',
+    reviewText: generateRandomReview(100),
     date: new Date(),
-    name: random_names[getRandomInt(0, random_names.length)],
+    name: randomNames[getRandomInt(0, randomNames.length)],
     rating: {
       cleanliness: getRandomInt(1, 5),
```

Comment:
Ratings shouldnt be prime int's they can be floats (1.56) you could consider changing your random function to generate a string and concat on the decimal (random(1,5) + '.' + random(1,99)); Or think of a different way to accomplish this.

Reply:
https://github.com/keybox130/Reviews-Service/pull/1/files/0b9b2ebdee1f09bca326960678f7fc69a9b5ba3c..014a120f3f4739003f7d084f60d9fe05b115aaec#r495199422